redwood-layout-composeui/src/main/kotlin/app/cash/redwood/layout/ComposeColumn.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.redwood.layout
+
+import androidx.compose.runtime.Composable
+import app.cash.redwood.LayoutModifier
+import app.cash.redwood.flexcontainer.FlexDirection
+import app.cash.redwood.layout.api.CrossAxisAlignment
+import app.cash.redwood.layout.api.MainAxisAlignment
+import app.cash.redwood.layout.api.Overflow
+import app.cash.redwood.layout.api.Padding
+import app.cash.redwood.layout.widget.Column
+import app.cash.redwood.widget.Widget
+
+internal class ComposeColumn : Column<@Composable () -> Unit> {
```

Comment:
`ComposeColumn` and `ComposeRow` are extremely similar to `ViewColumn` and `ViewRow`. Going to hold off combining them for now in case the `UiKit` variants don't follow the same pattern.

Reply:
It may not even be worth it since they're just some glue